### DISCUSSION TITLE
Visible white flash switching page in dark mode

### DISCUSSION THREAD

**Discussion by dvarrazzo** [pradyunsg/furo#230] [Potential Issue] [answered]
If you are in dark mode and click a link to another page, there is a white flash before the dark theme is applied.

This defeats the purpose of the dark theme as a way to relax your eyes: being flashed when your pupils are dilated is not a nice experience.

**Comment by pradyunsg** [comment, marked as answer]
I am unable to reproduce this, on Firefox, Chrome or Safari on MacOS.

Please provide clear reproduction steps and a video of this behaviour if possible.

**Reply by dvarrazzo** [reply]
Ok, I'll try and let you know. I use firefox. Maybe it's interacting with some extension...

**Reply by pradyunsg** [reply]
Okay, on really long pages, I've been able to reproduce this.

**Reply by pradyunsg** [reply]
Fixed and released, in the first stable release of Furo -- Aspiring Avocado. :)

**Reply by dvarrazzo** [reply]
It seems it fixed the problem for me too. Thank you
